javascript-practice/src/js/constants/constant.js

```diff
@@ -0,0 +1,15 @@
+export const databaseURL =
+  'https://javascript-training-81f7a-default-rtdb.asia-southeast1.firebasedatabase.app';
+export const regex =
```

Comment:
1. Split your constant files
2. Regex for what?
3. Not same format name for constant

Reply:
What do you mean "split constant file"? Is that this file divided into 2 files?

Reply:
Fixed.